pinot-core/src/main/java/org/apache/pinot/core/auth/Actions.java

```diff
@@ -0,0 +1,187 @@
+    public static final String LIST_TASK_TYPES = "ListTaskTypes";
+    public static final String LIST_TASK_QUEUES = "ListTaskQueues";
+    public static final String GET_TASK_QUEUE_STATE = "GetTaskQueueState";
+    public static final String LIST_TASKS = "ListTasks";
+    public static final String GET_TASK_METADATA = "GetTaskMetadata";
+    public static final String DELETE_TASK_METADATA = "DeleteTaskMetadata";
+    public static final String GET_TASK_COUNTS = "GetTaskCounts";
+    public static final String DEBUG_TASKS = "DebugTasks";
+    public static final String DEBUG_TASK = "DebugTask";
+    public static final String GET_TASK_STATES = "GetTaskStates";
+    public static final String GET_TASK_STATE = "GetTaskState";
+    public static final String GET_TASK_CONFIG = "GetTaskConfig";
+    public static final String GET_ALL_TASK_STATES = "GetAllTaskStates";
+    public static final String GET_SCHEDULER_INFO = "GetSchedulerInfo";
+    public static final String SCHEDULE_TASK = "ScheduleTask";
+    public static final String SCHEDULE_TASKS = "ScheduleTasks";
+    public static final String EXECUTE_TASK = "ExecuteTask";
+    public static final String CLEANUP_TASK = "CleanupTask";
+    public static final String CLEANUP_TASKS = "CleanupTasks";
+    public static final String STOP_TASKS = "StopTasks";
+    public static final String RESUME_TASK = "ResumeTasks";
+    public static final String UPDATE_TASK_QUEUE = "UpdateTaskQueue";
+    public static final String DELETE_TASK = "DeleteTask";
+    public static final String DELETE_TASKS = "DeleteTasks";
+    public static final String DELETE_TASK_QUEUE = "DeleteTaskQueue";
+    public static final String CREATE_TENANT = "CreateTenant";
+    public static final String UPDATE_TENANT = "UpdateTenant";
+    public static final String DELETE_TENANT = "DeleteTenant";
+    public static final String LIST_TENANT = "ListTenant";
+    public static final String LIST_TENANTS = "ListTenants";
+    public static final String GET_TENANT_METADATA = "GetTenantMetadata";
+    public static final String UPDATE_TENANT_METADATA = "UpdateTenantMetadata";
+    public static final String ESTIMATE_UPSERT_MEMORY = "EstimateUpsertMemory";
+    public static final String GET_VERSION = "GetVersion";
+    public static final String GET_TABLE_CONFIGS = "GetTableConfigs";
+    public static final String GET_ZNODE = "GetZnode";
+    public static final String GET_ZNODES = "GetZnodes";
+    public static final String DELETE_ZNODE = "DeleteZnode";
+    public static final String UPDATE_ZNODE = "UpdateZnode";
+    public static final String LIST_ZNODE = "ListZnode";
+    public static final String LIST_ZNODES = "ListZnodes";
+    public static final String UPDATE_QPS = "UpdateQPS";
+    public static final String GET_FIELD_SPEC = "GetFieldSpec";
+  }
+
+  // Action names for table
+  public static class Table {
+    public static final String ADD_TABLE = "AddTable";
+    public static final String VALIDATE = "Validate";
```

Comment:
This is confusing. Let's rename the config related actions:
<GET|UPDATE|VALIDATE|DELETE>TableConfig - TableConfig only
<GET|UPDATE|VALIDATE|DELETE>Schema - Schema only
<GET|UPDATE|VALIDATE|DELETE>TableConfigs - TableConfig and Schema (`TableConfigs` class is just a wrapper over table config and schema)

Reply:
There are two TableConfig related API, one at cluster level to get all table configs, and other at table level, are you suggesting to change them to TableConfig?

Are you suggesting to remove GetSchema, DeleteSchema, UpdateSchema, etc. and use Schema action for all of them?

Reply:
No. What I meant is that `Validate` is not accurate. We should use `ValidateTableConfig`, `ValidateSchema`, `ValidateTableConfig`, instead of `Validate` because the target is not included